src/content/en/updates/2017/02/media-session.md

```diff
@@ -0,0 +1,374 @@
+      // User clicked "Previous Track" media notification icon.
+      index = (index - 1 + playlist.length) % playlist.length;
+      playAudio();
+    });
+    
+    navigator.mediaSession.setActionHandler('nexttrack', function() {
+      // User clicked "Next Track" media notification icon.
+      index = (index + 1) % playlist.length;
+      playAudio();
+    });
+
+    function playAudio() {
+      audio.src = playlist[index];
+      audio.play()
+      .then(_ => { /* Set up Media Session... */ })
+      .catch(error => { console.log(error); });
+    }
+
+    playButton.addEventListener('pointerup', function(event) {
+      playAudio();
+    });
+
+The Media Session API allows you as well to show "Seek Backward" and "Seek
+Forward" media notification icons if you want to control the amount of time
+skipped.
+
+    let skipTime= 10; /* Time to skip in seconds */
+
+    navigator.mediaSession.setActionHandler('seekbackward', function() {
+      // User clicked "Seek Backward" media notification icon.
+      audio.currentTime = Math.max(audio.currentTime - skipTime, 0);
+    });
+    
+    navigator.mediaSession.setActionHandler('seekforward', function() {
+      // User clicked "Seek Forward" media notification icon.
+      audio.currentTime = Math.min(audio.currentTime + skipTime, audio.duration);
+    });
+
+Note: As the browser may consider the web app is not be playing when media
+files are seeking or loading, you can override this behaviour by setting
+`navigator.mediaSession.playbackState` to `"playing"` or `"paused"`.  This
+comes handy when you want to make sure your web app UI state is synced with the
+media notification controls.
+
+The cool thing about the Media Session API is that the notification tray is not
+the only place where media metadata and controls are visible. The media
+notification is synced automagically to any paired wearable device. And it also
+shows up on lock screens.
+
+<div class="clearfix"></div>
+<div class="attempt-left">
+  <figure>
+    <img src="https://placehold.it/350x350?text=Lock screen" alt="TODO">
+    <figcaption>Lockscreen</figcaption>
+  </figure>
+</div>
+<div class="attempt-right">
+  <figure>
+    <img src="https://placehold.it/350x350?text=Wear notification" alt="TODO">
+    <figcaption>Wear</figcaption>
+  </figure>
+</div>
+<div class="clearfix"></div>
+
+## Make it play nice offline
+
+I know what you're thinking now... *[Service Worker] to the rescue!*
+
+True but first and foremost, you want to make sure **all items in this
+checklist are checked**:
+
+1. All media and artwork files are served with the appropriate `Cache-Control`
+HTTP header. This will allow the browser to cache and reuse previously fetched
+resources. See the [Caching checklist].
+2. Make sure all media and artwork files are served with the
+`Allow-Control-Allow-Origin: *` HTTP header. This will allow third-party web
+apps to fetch and consume HTTP responses from your web server.
```

Comment:
Do we want to advertise `*` or should we tell web developers that listing their website's origin is preferable?